src/controllers/mainController.ts

```diff
@@ -1646,6 +1647,7 @@ export default class MainController implements vscode.Disposable {
             if (editor.document.getText(selectionToTrim).trim().length === 0) {
                 return;
             }
+            store.delete(uri);
```

Comment:
Do we clean up store/memory usage when the user closes a query tab?

Reply:
https://github.com/microsoft/vscode-mssql/pull/18602/files#diff-226c7e3c205fb1d29e63f899a152924e487bf890bb934ecf2542a739504d2a08R2103 Added here